### PR TITLE
Add tlsVerify flag 

### PR DESCRIPTION
**Pull request by Sajadorouji**
Sometimes our urls doesn't have any valid ssl certificate

### CODE REVIEW

File: cmd/linkchecker/linkchecker.go [hunk 1]
```diff
@@ -37,6 +39,7 @@ var (
 
 func main() {
 	flag.StringVar(&host, "host", "", "Hostname and port of site to check.")
+	flag.StringVar(&tlsVerify, "tlsVerify", "true", "If you want to try site with invalid certificate, default: true")
```
Comment (johndopazo) [inline cmd/linkchecker/linkchecker.go]: Did you consider using `flag.BoolVar` https://golang.org/pkg/flag/#BoolVar?

Then the usage might be something like this.

```
./linkchecker -host=www.ardanstudios.com -skiptls
```

```
flag.StringVar(&skipTLS, "skiptls", false, "To try site with invalid certificate, default: false")

...

http.DefaultTransport.(*http.Transport).TLSClientConfig = &tls.Config{InsecureSkipVerify: skipTLS}
```

**Review by johndopazo — COMMENTED** [review]
Thank you so much for your pull request!

File: cmd/linkchecker/linkchecker.go (hunk 1 above)
Comment (Sajadorouji) [reply cmd/linkchecker/linkchecker.go]: Thanks for the suggestion, changed it.

**Review by johndopazo — APPROVED** [review]
Looks good to me!

**Pull request merged**